src/component/PortofolioSection/PortofolioSection.tsx

```
@@ -0,0 +1,92 @@
+          onClick={nextImage}
+        >
+          <h2>MY PREVIOUS WORKS</h2>
+          <img src={content[index].image} alt={content[index].title} />
+          <div className={styles['portofolio__detail']}>
+            <span className={styles['portofolio__detail__span']} />
+            <div className={styles['portofolio__detail__desc']}>
+              <h3>{content[index].title}</h3>
+              <br />
+              {/* <div className="portofolio__techStack"></div> */}
+              {content[index].description instanceof Array ? (
+                content[index].description.map((desc, idx) => (
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)